### PR TITLE
LOOFAH-31 Add Crafts to model and implement API queries to fetch them

### PR DESCRIPTION
**Pull request by Learner5200**

### CODE REVIEW

File: src/main/java/com/loofah/graph/api/config/DatabaseSeeder.java [hunk 1]
```diff
@@ -1,36 +1,55 @@
...
 
-        skillRepository.save(Skill.builder().withId("1").withTitle("title1").withDescription( "description1").withCategoryId("1").withExamples(Collections.EMPTY_LIST).build());
-        skillRepository.save(Skill.builder().withId("2").withTitle("title2").withDescription( "description2").withCategoryId("1").withExamples(Collections.EMPTY_LIST).build());
-        skillRepository.save(Skill.builder().withId("3").withTitle("title3").withDescription( "description3").withCategoryId("2").withExamples(Collections.EMPTY_LIST).build());
-
-        categoryRepository.save(Category.builder().withId("1").withTitle("categoryTitle1").withDescription("categoryDescription1").build());
-        categoryRepository.save(Category.builder().withId("2").withTitle("categoryTitle2").withDescription("categoryDescription1").build());
+        SKILLS.forEach(skill -> skillRepository.save(skill));
+        CATEGORIES.forEach(category -> categoryRepository.save(category));
+        CRAFTS.forEach(craft -> craftRepository.save(craft));
```
Comment (vijamesDD) [inline src/main/java/com/loofah/graph/api/config/DatabaseSeeder.java]: So simple and neat 👏 LOVE IT

File: src/main/java/com/loofah/graph/api/services/GraphQLProvider.java [hunk 2]
```diff
@@ -38,18 +34,22 @@
...
+    private CraftsQuery craftsQuery;
+    private CraftQuery craftQuery;
 
     @Autowired
     public GraphQLProvider(SkillsQuery skillsQuery,
                            SkillQuery skillQuery,
                            CategoryQuery categoryQuery,
                            CategoriesQuery categoriesQuery,
-                           SkillsByCategoryQuery skillsByCategoryQuery){
+                           SkillsByCategoryQuery skillsByCategoryQuery, final CraftsQuery craftsQuery, final CraftQuery craftQuery){
```
Comment (vijamesDD) [inline src/main/java/com/loofah/graph/api/services/GraphQLProvider.java]: [tiny] could you put these on separate lines for consistency pls?

**Review by vijamesDD — APPROVED** [review]

File: src/main/java/com/loofah/graph/api/config/DatabaseSeeder.java (hunk 1 above)
Comment (elishkaflint) [reply src/main/java/com/loofah/graph/api/config/DatabaseSeeder.java]: agree - this is great :)

File: src/test/java/com/loofah/graph/api/integrationTests/IntegrationTest.java [hunk 3]
```diff
@@ -55,7 +46,7 @@
     @Value("classpath:testQueries/skillsQuery.txt")
     private Resource skillsQuery;
 
-    @Value("classpath:testQueries/CategoryQuery.txt")
+    @Value("classpath:testQueries/categoryQuery.txt")
```
Comment (elishkaflint) [inline src/test/java/com/loofah/graph/api/integrationTests/IntegrationTest.java:49]: 🙌

**Review by elishkaflint — APPROVED** [review]
awesome job @Learner5200

File: src/main/java/com/loofah/graph/api/services/GraphQLProvider.java (hunk 2 above)
Comment (Learner5200) [reply src/main/java/com/loofah/graph/api/services/GraphQLProvider.java]: Good spot, done now :)

**Pull request merged**